web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -87,8 +88,12 @@ export class MainPageComponent implements OnInit {
   }
 
   private showEditLayerDialog(layerId: string) {
-    this.dialog.open(LayerDialogComponent, {
-      data: { layerId },
-    });
+    this.subscription.add(
+      this.activeProject$.pipe(take(1)).subscribe(project => {
```

Comment:
I did a [little research](https://medium.com/@benlesh/rxjs-dont-unsubscribe-6753ed4fda87), and you can do this to have the subscription automatically disposed of for you after one item is emitted:
```
this.activeProject$.take(1).subscribe(project => this.dialog.open...);
```

`this.subscription.add` can the be removed. Note that `{}` is not needed in the lambda in this case.